src/lib/layout/layout.ts

```diff
@@ -97,6 +98,7 @@ interface Options {
    * The place to output the build, relative to project root.
    */
   buildOutputRelative?: string
+  entrypointPath?: string | null
```

Comment:
why `| null`?

Reply:
Removed it. Was there because I don't link returning `undefined` in functions (`normalizeEntrypoint`)

Reply:
Missed the connection, fwiw agree!